lib/src/random_bytes.dart

```diff
@@ -0,0 +1,17 @@
+// Copyright (c) 2015, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+import 'dart:math';
+import 'dart:typed_data';
+
+/// Generates a cryptographically secure random UInt8List of [length] bytes
+Uint8List randomBytes(int length) {
```

Comment:
What is the use-case for this particular function?
It uses secure random, which is not as cheap as non-secure randomness.
Consider taking an optional `Random` instance to use, with `Random.secure()` as default if you pass nothing.
```dart
Uint8List randomBytes(int length, {Random random}) {
  random ??= Random.secure();
  ...
}
```

Would it make sense to have a `void fillWithRandomBytes(Uint8List target, [int start = 0, int end])` instead or as well? 
That could allow you to reuse an existing buffer, or add random bytes inside a larger structure, without allocating a new intermediate `Uint8List`.

Reply:
The function is inspired by browsers randomBytes. People use it there to generate SALT or nonces. As a result it should be (same as the randomBytes functions in browsers) be by default usable for cryptographic purposes. Otherwise the name would be probably be missleading.